src/SFA.DAS.AssessorService.Data.IntegrationTests/Repositories/FixtureBase.cs

```diff
@@ -0,0 +1,120 @@
+﻿using SFA.DAS.AssessorService.Data.IntegrationTests.Handlers;
+using SFA.DAS.AssessorService.Data.IntegrationTests.Models;
+using SFA.DAS.AssessorService.Domain.Consts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.AssessorService.Data.IntegrationTests.Repositories
```

Comment:
We can use file-level namespaces to save a level of indentation. If you type a semicolon at the end of this line, it will remove the next block of indentation so that the namespace declaration doesn't use an indent all by itself.

Reply:
File level namespaces are the latest thing agreed, and I would be content to introduce them on a new project which used them everywhere - however I don't like them at all - I like the indentation and I certainly don't want to see a mixed use of them on a legacy project - as it makes me feel queasy - so please don't use them in assessor